test/app.e2e-spec.ts

```diff
@@ -1,7 +1,7 @@
 import { Test, TestingModule } from '@nestjs/testing';
 import { INestApplication } from '@nestjs/common';
 import * as request from 'supertest';
-import { AppModule } from './../src/app.module';
+import { AppModule } from '../src/app.module';
```

Comment:
Prefer relative imports, please take a look at your IDE settings and tsconfig.json. You shoul be able to import this module by writting.
```ts
import {AppModule} from 'src/app.module'
```